src/extension_consts.ml

```diff
@@ -0,0 +1,15 @@
+let ocaml_prefixed key = "ocaml." ^ key
+
+module Commands = struct
+  let select_sandbox = ocaml_prefixed "select-sandbox"
+
+  let restart_language_server = ocaml_prefixed "server.restart"
+
+  let select_sandbox_and_open_terminal = ocaml_prefixed "open-terminal-select"
+
+  let open_terminal = ocaml_prefixed "open-terminal"
+
+  let switch_impl_intf = ocaml_prefixed "switch-impl-intf"
+end
+
+(* TODO: add other constants such as settings keywords etc *)
```

Comment:
I have a strong dislike for modules that just group unrelated constants. I prefer to encapsulate constants in logical modules as much as possible. I realize that we haven't come up with a good alternative here yet, so this shouldn't block the PR from being merged. We definitely don't want to make the situation worse in the future.

Reply:
I also have mixed feelings about non-logically related constants, but 
1. This may be a good thing, e.g., almost all android applications extract out strings in a separate file, so that user-visible stuff such as warning messages can be easily changed (+ i18n but not our case :-))
2. I don't have alternatives either as you say :-)